content/docs/0.16.x/quality_gates/get_started/index.md

```diff
@@ -0,0 +1,128 @@
+
+The rest of this section assumes you have Keptn [installed](../../operate/install/) on your cluster, your Keptn CLI is [authenticated](../../operate/install/#authenticate-keptn-cli), and you have a deployed application that is monitored by your favorite monitoring solution.
+
+Given these requirements, this section defines the entities of a Keptn project, stage, and services using an example. Afterwards, four steps explain how to set up the quality gate, trigger it, and finally see the quality gate evaluation result.
+
+## Definition of project, stage, and service
+
+Let's assume you have an application that is running in an environment and composed of one or multiple services (aka. microservices). For example, you have an application called `easyBooking`, which can be broken down into the `booking` and `payment` service. Besides, the application is running in a `quality_assurance` environment (aka. stage).
+In order to manage your application and services in Keptn, you need a Keptn [project, stage](../../manage/project/) and [services](../../manage/service).
+For the `easyBooking` application, the Keptn entities of a project, stage, and service map to the example as follows:
+
+* `project`: *easyBooking*
+* `stage`: *quality_assurance*
+* `service`: *booking* & *payment* (For the sake of simplicity, a quality gate will be configured for `booking` only.)
+
+For defining the stage(s) a service has to go through, a [Shipyard](../../manage/shipyard) file is needed. Since a quality gate should be configured for the *quality_assurance* environment only, the corresponding Shipyard for the *easyBooking* project looks as follows:
+
+```yaml
+apiVersion: "spec.keptn.sh/0.2.3"
+kind: "Shipyard"
+metadata:
+  name: "shipyard-sockshop"
+spec:
+  stages:
+    - name: "quality-assurance"
+```
+
+<details><summary>*Why I do not need to specify a task sequence for quality gates in this stage?*</summary>
+<p>
+
+You do not have to define any task sequence in a stage because quality gates (aka. `evaluation`) are a built-in Keptn capability. Hence, there is no need to explicitly add an `evaluation` sequence. However, the explicit form of the above Shipyard file would look as the following one, which behaves the same way: 
+
+```yaml
+apiVersion: "spec.keptn.sh/0.2.3"
+kind: "Shipyard"
+metadata:
+  name: "shipyard-sockshop"
+spec:
+  stages:
+    - name: "quality-assurance"
+      sequences:
+       - name: "evaluation"
+         tasks:
+         - name: "evaluation"
+```
+
+</p>
+</details>
+
+## Create project and service
+
+* To create the Keptn project `easyBooking`, use the [keptn create project](../../reference/cli/commands/keptn_create_project/) command.
+
+* To create the Keptn service `booking`, use the [keptn create service](../../reference/cli/commands/keptn_create_service/) command.
+
+**Note:** It is not needed to create the stage since it is declared in the Shipyard that is applied during the project creation. 
```

Comment:
```suggestion
**Note:** It is not necessary to create the stage since it is declared in the Shipyard that is applied during the project creation. 
```